gendiff/core.py

```diff
@@ -1,16 +1,14 @@
-from typing import Any
-
-from gendiff.diff import get_diff_view
 from gendiff.file_operations import parse_file
 from gendiff.formatting import get_diff_str
+from gendiff.string_templates import get_diff_str_nested
 
 
 def generate_diff(
     first_file: str, second_file: str, output_format: str = "stylish"
-) -> Any:
+) -> str:
     f1 = parse_file(first_file)
     f2 = parse_file(second_file)
     if output_format == "stylish":
-        return get_diff_view(f1, f2)
+        return "{\n" + get_diff_str_nested(f1, f2) + "}"
```

Comment:
a function that generates diff should create `"{\n"` and `"}"` on its own - so you'll just call it to get diff string